lighthouse-cli/bin.ts

```diff
@@ -229,7 +229,7 @@ function handleError(err: LightHouseError) {
   if (err.code === 'ECONNREFUSED') {
     showConnectionError();
   } else if (err.message.toLowerCase().includes('multiple tabs')) {
```

Comment:
no need for these 2 lines anymore :) it will just go inside the else. (thanks for cleaning up)